cc_plugin_glider/glider_dac.py

```diff
@@ -31,7 +31,9 @@ class GliderCheck(BaseNCCheck):
     _cc_display_headers = {3: "Required", 2: "Recommended", 1: "Suggested"}
     acceptable_platform_types = {"Seaglider", "Spray Glider", "Slocum Glider"}
 
-    def __init__(self):
+    def __init__(self, options=None):
```

Comment:
@jr3cermak do you want a generic `options` kw or should we name this `skip_variables`? I may not be understanding what you want to do but it seems that it is the latter, right?